llvm/lib/Transforms/IPO/DynUntrustedAllocPost.cpp

```diff
@@ -37,6 +37,7 @@
 #include "llvm/Support/JSON.h"
 #include "llvm/Support/MemoryBuffer.h"
 #include "llvm/Support/Path.h"
+#include "llvm/Transforms/Utils/Local.h"
 
```

Comment:
why no cl::opt to pass a profile for testing?

Reply:
Added two cl::opts. One for the profile path, the other for removing hooks.